packages/fxa-admin-panel/src/components/EmailBlocks/Account/index.tsx

```diff
@@ -46,7 +53,10 @@ export const ClearButton = ({
       return;
     }
 
-    clearBounces({ variables: { email } });
+    // This could be improved to clear bounces for individual email
+    // addresses, but for now it seems satisfactory to clear all bounces
+    // for all emails, since they own all of the addresses
+    emails.forEach((email) => clearBounces({ variables: { email } }));
```

Comment:
Note, currently clicking "Clear all bounces" doesn't clear bounces for secondary addresses. This should hopefully do the trick.

Reply:
@bbangert this is what I was referring to about executing on each address.

Reply:
PR LGTM besides a possible improvement here, we could pull this out and only clear the primary e-mail for now. Would say maybe hold until @bbangert chimes in here but I'll r+ since this isn't a blocker for this PR from my PoV, nice work @jodyheavener, the highlight is sweet!

Reply:
I see no reason not to clear them all. 👍